src/components/ebay-menu/component.js

```diff
@@ -2,9 +2,11 @@ const assign = require('core-js-pure/features/object/assign');
 const findIndex = require('core-js-pure/features/array/find-index');
 const scrollKeyPreventer = require('makeup-prevent-scroll-keys');
 const rovingTabindex = require('makeup-roving-tabindex');
+const typeahead = require('makeup-typeahead');
 const eventUtils = require('../../common/event-utils');
 const menuUtils = require('../../common/menu-utils');
-const NodeListUtils = require('../../common/nodelist-utils');
+
+const TYPEAHEAD_TIMEOUT_LENGTH = 1300;
```

Comment:
@ianmcburnie Would we want to allow this value to be overridden by teams?
If so we can do this where we read the value: `this.input.typeaheadTimeoutLength || TYPEAHEAD_TIMEOUT_LENGTH;`

Reply:
I don't think we'd want them to override.